src/lib/space-map-image.ts

```diff
@@ -0,0 +1,29 @@
+import type { Space } from "~/types/Space";
+
+/**
+ * Create a static map of building map with space marker
+ *
+ * @return image of building map with space marker
+ */
+export default function (space: Space, mapboxToken: string) {
+  const { bounds } = space.building;
+  // Use invisible markers at the bounds of the building
+  // and set zoom to `auto` so map extends exactly to building
+  const invisibleMarkerUrl = encodeURIComponent(
+    "https://5d4b289a581fb6255e317d9d--assets-jasper.netlify.com/no-marker.png"
```

Comment:
We should really move this asset inside the project :)

Reply:
We can move it to the `public` folder and concat it with `env.URL`, but it would mean it will stop working at dev, right? No big deal of course, it's only used for `og:image`. We might also hardcode/configure the public base URL and always refer to the image in prod. Which option you like more?